src/disco/quic/fd_quic.c

```diff
@@ -306,11 +306,10 @@ before_frag( void * _ctx,
   int handled_port = dst_port == ctx->legacy_transaction_port ||
                      dst_port == ctx->quic->config.net.listen_udp_port;
 
+  /* Ignore traffic e.g. for shred tile */
   if( FD_UNLIKELY( !handled_port ) ) {
-    FD_LOG_ERR(( "Firedancer received a UDP packet on port %hu which was not expected. "
```

Comment:
We want to keep this check at the moment to verify the XDP traffic is right. Can you plumb shred port through and add it to the check here